### DISCUSSION TITLE
Why is LBP3 on PS4/5 considered incompatible?

### DISCUSSION THREAD

**Discussion by Kee7702** [LBPUnion/ProjectLighthouse#641] [Q&A]
It is the inability to easily patch the EBOOT like you can on PS3, actual differences in the requests made to the server, or some other factor I'm not considering?

Has any actual testing been done with attempting to patch to a custom game server, or provide a custom certificate so custom dns will work?

**Comment by vilijur** [comment]
You need to be able to log into PSN to go online. And you can only log into PSN on the latest system software on PS4/5, which can't be done on a jailbroken PS4 without losing the jailbreak. Also, the official servers still exist on PS4/5, so there's no need to attempt to make Project Lighthouse compatible with LBP3 on PS4/5.

**Reply by TorutheRedFox** [reply]
-yet

**Reply by sudokoko** [reply]
> -yet

We'll come to that road if and when we need to. However there's no real point to investing time and resources into this if it's not something that's currently viable or necessary.

**Reply by Kee7702** [reply]
I do understand that PSN is a requirement much like it is on PS3 (though on PS3 it's extremely easy to bypass updates whereas on PS4 it's a bit more complicated). I wasn't asking why it wasn't considered compatible, but what testing makes it strictly incompatible?

**Reply by TorutheRedFox** [reply]
> I do understand that PSN is a requirement much like it is on PS3 (though on PS3 it's extremely easy to bypass updates whereas on PS4 it's a bit more complicated). I wasn't asking why it wasn't considered compatible, but what testing makes it strictly incompatible?

probably the same stuff that makes PS3 incompatible with the current official servers now (except for the root cert change)

**Reply by sudokoko** [reply]
> > I do understand that PSN is a requirement much like it is on PS3 (though on PS3 it's extremely easy to bypass updates whereas on PS4 it's a bit more complicated). I wasn't asking why it wasn't considered compatible, but what testing makes it strictly incompatible?
> 
> probably the same stuff that makes PS3 incompatible with the current official servers now (except for the root cert change)

PS3 is incompatible because SCEE denies any PS3 client from connecting. There are various ways they could do this. Since you still get a 403 when attempting to connect to official servers, the servers are still "online" but you can't connect to them. The server URL did not change between PS3 and PS4.
